cime_config/testdefs/testlist_clm.xml

```diff
@@ -195,7 +195,7 @@
       <option name="comment"  >CTSM science test list for the half degree grid with latest CLM version for historical</option>
     </options>
   </test>
-  <test name="SMS_Ln9" grid="C96_t061" compset="I2000Clm60Sp" testmods="clm/clm60cam7LndTuningMode--clm/nofireemis">
+  <test name="SMS_Ln9" grid="C96_t232" compset="I2000Clm60Sp" testmods="clm/clm60cam7LndTuningMode--clm/nofireemis">
```

Comment:
`ERROR: no alias C96_t232 defined`

Options:
1) Back this change out and file an issue.
2) Modify submodule file ccs_config/modelgrid_aliases_nuopc.xml

Reply:
Hmmm. It looks like the alias is in the latest ccs_config. But, must not be in the one we are currently using for CTSM. So we could figure out what version of ccs_config needs to be used here and also update that. Or remember to do this as part of the externals update that needs to happen as well. @slevis-lmwg do you have a preference there?

Reply:
Seems riskier to update the submodule now. It could open a can of worms, we just don't know...

I prefer that we add a checkbox about updating the C96_t061 test wherever the list of checkboxes is for the next submodules update.